openslides/utils/access_permissions.py

```diff
@@ -85,16 +85,15 @@ def add_collection_string(
         """
         user_ids: Set[int] = set()
 
-        all_full_data = await element_cache.get_all_full_data()
         for collection_string in collection_strings:
+            full_data = await element_cache.get_collection_full_data(collection_string)
```

Comment:
Maybe plese rename this to something like "collection_full_data".

Reply:
done